frontend/src/lib-customizations/defaults/citizen/i18n/sv.tsx

```diff
@@ -368,6 +368,9 @@ const sv: Translations = {
       selectedChildren: 'Utvalda barn',
       selectChildrenInfo:
         'Anmäl endast heldagsfrånvaro. Kortare frånvaro kan anmälas genom reservationer.',
+      lockedAbsencesWarningTitle: 'Frånvaro i flera dagar',
+      lockedAbsencesWarningText:
+        'Du håller på att markera frånvaro för dagar ,där anmälningstiden har gått ut. Detta kan endast återkallas genom att vara i kontakt med personalen.',
```

Comment:
```suggestion
        'Du håller på att markera frånvaro för flera dagar, där anmälningstiden har gått ut. Detta kan endast återkallas genom att vara i kontakt med personalen.',
```

Comma in wrong place and missing `flera`.